packages/ember-glimmer/lib/syntax/curly-component.js

```diff
@@ -89,7 +89,7 @@ class CurlyComponentManager {
       bucket.classRef = args.named.get('class');
     }
 
-    assert('classNameBindings must not have spaces in them', () => {
+    assert('classNameBindings must not have spaces in them', (() => {
```

Comment:
Can you add `component.toString()` in this message? I submitted https://github.com/emberjs/ember.js/pull/13685 to do this, but no need to land that and cause a rebase conflict...